README.md

```diff
@@ -13,9 +13,36 @@ whether this method is more or less (or equally) likely to pick a good pivot
 compared to simply choosing the first element. Assume that all permutations are
 equally likely, i.e. the input array is ordered randomly.
 
+I used the lecture slides/video on sorting, quicksort to look at slide 34 to review the material and compare my answer. Ali Torabi showed me a link https://walkccc.me/CLRS/Chap07/Problems/7-5/ to understand the math side of this and helped me. I went to his office hours where I learned I did not show all the possibilities which I then fixed. 
+
+I certify that I have listed all sources used to complete this exercise, including the use of any Large Language Models. All of the work is my own, except where stated otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is suspected, charges may be filed against me without prior notice.
+
 Your answer must derive probabilities for choosing a good pivot and
 quantitatively reason with them.
 
+When choosing 3 random elements, the array is essentially split into 3 parts, left, middle, and right. The middle includes the best pivots which is about 1/2 of the array, and the left and right sides are each 1/4 probabilty of holding the pivot. While looking at this, let's look at the $3^3$ = 27 possibilities.
+1. Elements are chosen left, middle, right = $(1/4) * (1/2) * (1/4)$ = $(1/32)$
+   a.) Three ways it can switch around (LMR, MLR, LRM) so $(1/32) * 3 = 3/32 = 6/64$
+2. Elements are chose left, left, middle = $(1/4) * (1/4) * (1/2)$ = $(1/32)$
+   a.) Three ways it can switch around (LLM, LML, MLL) so $(1/32) * 3 = 3/32 = 6/64$
+3. Elemenets are chosen middle, right, right = $(1/2) * (1/4) * (1/4)$ = $(1/32)$
+   a.) Three ways it can switch around (MRR, RMR, RRM) so $(1/32) * 3 = 3/32 = 6/64$
+4. Elements are chosen left, left, left or right right right = $(1/4) * (1/4) * (1/4)$ = $(1/64)$
+   a.) Since this can either be LLL or RRR = $(1/32) * 2 = (2/32) = 4/64$
+5. Elements are chosen left, left, right = $(1/4) * (1/4) * (1/4)$ = $(1/64)$
+   a.) Three ways it can switch around (LLR, LRL, RLL) so $(1/32) * 3 = 3/32 = 6/64$
+6. Elements are chosen right, right, left = $(1/4) * (1/4) * (1/4)$ = $(1/64)$
+    a.) Three ways it can switch around (RRL, RLR, LRR) so $(1/32) * 3 = 3/32 = 6/64$
+
+So, 
+$(6/64) + (6/64) + (6/64) + (4/64) + (6/64) + (6/64) = 34/64$ 
```

Comment:
I think you're missing some good cases here...

Reply:
Yes, sorry, still working on this.